bigchaindb/consensus.py

```diff
@@ -139,12 +139,21 @@ def validate_transaction(bigchain, transaction):
                 raise exceptions.OperationError(
                     'Only federation nodes can use the operation `CREATE`')
 
+            # validate digital asset
+            # A create transaction will have only one output
+            asset = transaction['transaction']['asset']
+            assets.validate_asset_creation(asset['data'], asset['divisible'], asset['updatable'],
+                                           asset['refillable'], transaction['transaction']['conditions'][0]['amount'])
+            # TODO: Validate uuid?
```

Comment:
This is only important at creation time. To things can happen:
- The asset_id is a duplicate
- The asset_id is not a valid uuid in canonical form
After that all nodes independently check that the asset_id matches the inputs.

I think that this would be solved once we improve the asset creation where a client would contact several nodes instead on only one.